recipe_scrapers/spisbedre.py

```diff
@@ -28,7 +29,11 @@ def total_time(self):
         return self.schema.total_time()
 
     def yields(self):
-        return self.schema.yields()
+        servings = int(self.recipe_json.get("serving_size"))
+        yield_type = (
+            "item" if self.recipe_json.get("serving_size_unit_id") != 1 else "serving"
+        )
+        return f"{servings} {yield_type}{'s' if servings != 1 else ''}"
```

Comment:
```suggestion
        return self.schema.yields()
```

We can gather the yield info directly from the json without any custom logic!

Reply:
That's only partly true as spisbedre.dk only renders a numeric value for the recipeYield tag, which doesn't indicate if the yield is per person or a number of items.
From the get_yields function in _utils.py I gathered that yield should reflect either servings or items, I hope this is a correct assumption.

So, to get the exact indication, it's necessary to fetch the information from the JSON structure which isn't in the JSON-LD format hence the additional parsing.

The third test case (mentioned below) is a recipe for buns, and the yield is number of buns - I hope this makes sense.